### ISSUE TITLE
v1.41.1-1 is freezing my KDE desktop

### ISSUE THREAD

**Issue by js-d-coder** [VSCodium/vscodium#316]
**Describe the bug**
v1.41.1-1 is freezing my KDE desktop randomly.

**Please confirm that this problem is VSCodium-specific**
- [ ] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.
- [x] Can't confirm because I don't use Microsoft's VS Code.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
1. Open VSCodium for coding.
2. Use it for few hours
3. KDE is frozen now

**Expected behavior**
it should not freeze my KDE desktop. *v1.41.0 of this app did not have this issue.*

**Desktop (please complete the following information):**
Operating System: Manjaro Linux 
KDE Plasma Version: 5.17.4
KDE Frameworks Version: 5.65.0
Qt Version: 5.14.0
Kernel Version: 5.4.6-2-MANJARO
OS Type: 64-bit
Processors: 8 × Intel® Core™ i7-8550U CPU @ 1.80GHz
Memory: 15.4 GiB of RAM

**Additional context**
(help me provide more info if required)

Thanks a lot for this app.

**Comment by mucinoab** [comment]
I am having the exact same issue.

VSCodium: 1.41.1 
Operating System: Linux Mint 19.3 Tricia x86_64
Kernel Version: 5.4.5-050405-generic
Desktop environment: Cinnamon 4.4.8
Window manager: Mutter(Muffin)
CPU: Intel i7-6600U (4) @ 3.400GHz
Memory: 8 GiB of RAM

**Comment by talentlessguy** [comment]
Freezes on mine sometimes too, when deleting files

**Comment by js-d-coder** [comment]
I am facing this issue with Visual Studio Code too. I don't think this is vscodium specific.

**Comment by drajathasan** [comment]
Same with me. I just open vscodium for few minutes, and I have my PC is so slow. After that I check my system monitor and I have my CPU almost 100%. But when I use vscode isn't happen when I use vscodium.  I use Arch Linux with vscodium 1.41.1-2.

![Screenshot pada 2020-01-22 12-58-21](https://user-images.githubusercontent.com/38057222/72869588-60c11f00-3d18-11ea-924b-f637235bf952.png)

**Comment by WhiteR4bbitt** [comment]
Is there any process that was spiking? I am having issues with `rg.exe` and trying to figure out what is triggering it so much (I renamed the unpacked file, but my current loss is I can't use **Global Search**)

Not sure if its related, but this process was spiking my CPU and making `VSCodium` unbearable to use. When I get time in the next few days I am going to see if its an `extension` or `built-in` because having EVERYTHING off it ran with 0 problems (except lack of all the disabled features ..)

I read in various places it could be related to the built in `TypeScript` support, but that was old and supposed to have been resolved. Most issues retaining my problem says to use the `excludes` (especially for large projects) which mine isn't that big and is using `excludes`. I should maybe even turn my comment into its own issue, if unrelated (after debugging if its actually just a 3rd party extension or not).
